data/human/deep missions.txt

```diff
@@ -1626,14 +1626,16 @@ event "deep: arculus empty"
 		remove fleet "Remnant Transport"
 		remove fleet "Remnant Home Guard"
 		remove fleet "Korath Ember Waste Raid"
+		remove fleet "Kor Alaji Raid"
 
 event "deep: arculus restored"
 	system "Arculus"
 		add fleet "Large Remnant" 3000
 		add fleet "Small Remnant" 2200
 		add fleet "Remnant Transport" 2200
 		add fleet "Remnant Home Guard" 2000
-		add fleet "Korath Ember Waste Raid" 40000
+		add fleet "Korath Ember Waste Raid" 120000
```

Comment:
```suggestion
```
I know this happens in vanilla, but I don't think restoring after an event should change fleet spawns

Reply:
Right, I may have misinterpreted that mission.  This would be the proper restoration:

```suggestion
		add fleet "Korath Ember Waste Raid" 120000
		add fleet "Kor Alaji Raid" 60000
```